src/resources/views/create.blade.php

```diff
@@ -1,64 +1,41 @@
 @extends('backpack::layout')
 
 @section('header')
-	<section class="content-header">
-	  <h1>
-	    {{ trans('backpack::crud.add') }} <span class="text-lowercase">{{ $crud->entity_name }}</span>
-	  </h1>
-	  <ol class="breadcrumb">
-	    <li><a href="{{ url(config('backpack.base.route_prefix'), 'dashboard') }}">{{ trans('backpack::crud.admin') }}</a></li>
-	    <li><a href="{{ url($crud->route) }}" class="text-capitalize">{{ $crud->entity_name_plural }}</a></li>
-	    <li class="active">{{ trans('backpack::crud.add') }}</li>
-	  </ol>
-	</section>
+  <section class="content-header">
+    <h1>
+      {{ trans('backpack::crud.add_a_new') }} <span class="text-lowercase">{{ $crud->entity_name }}</span>
```

Comment:
Any reason to change to `add_a_new` (it seems orthogonal to the issue at hand plus it will make the add buttons longer)?